src/coreclr/gc/gc.cpp

```diff
@@ -783,6 +783,11 @@ class t_join
         return join_struct.n_threads;
     }
 
+    int get_join_lock()
+    {
+        return join_struct.join_lock;
```

Comment:
`svr::join_structure::join_lock` is a `VOLATILE(int)`, should this be a `VolatileLoadWithoutBarrier`?

Reply:
it should be... I'll change it.